src/Agent.Plugins/GitCliManager.cs

```diff
@@ -175,23 +175,34 @@ public async Task<int> GitFetch(AgentTaskPluginExecutionContext context, string
             string progress = reducedOutput ? string.Empty : "--progress";
```

Comment:
Shouldn't this be used instead of hardcoded `--progress` argument?

Reply:
Ehm, not sure, since this is not in the scope of my changes.

Reply:
As I can see it's fixed now, thanks